css/blocks/features.css

```diff
@@ -0,0 +1,39 @@
+.features {
+  padding: 94px 0 94px;
+}
```

Comment:
use the same approach in all sections - section padding is responsible for inner space between section and its' content